R/zchunk_LB132.ag_an_For_Prices_USA_C_2005.R

```diff
@@ -38,43 +39,236 @@ module_aglu_LB132.ag_an_For_Prices_USA_C_2005_DISABLED <- function(command, ...)
     L100.FAO_ag_Prod_t <- get_data(all_data, "L100.FAO_ag_Prod_t")
     FAO_USA_an_Prod_t_PRODSTAT <- get_data(all_data, "aglu/FAO_USA_an_Prod_t_PRODSTAT")
 
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # ===================================================
+    # Converting cotton back to primary equivalent (seed cotton)
+    # Seed cotton has no price in PRICESTAT. Need to derive its price from cotton lint and cottonseed
+    # Assigning a price for game meat so that OtherMeat is assigned a price
+    FAO_USA_ag_an_P_USDt_PRICESTAT %>%
+      select(-country.codes, -item.codes, -element, -element.codes) %>%
+      gather(year, price, -countries, -item) %>%
+      filter(item == "Seed cotton" | item == "Cotton lint" | item == "Cottonseed" | item == "Game meat" | item == "Cattle meat") %>%
+      # Modify item names to one word so that they can be used as column names when spreading item and doing calculations
+      mutate(item = sub(" ", "_", item)) %>%
+      spread(item, price) %>%
+      mutate(Seed_cotton = Cotton_lint * CONV_COTTON_LINT + Cottonseed * (1 - CONV_COTTON_LINT)) %>%
```

Comment:
This is definitely not how I expected CONV_COTTON_LINT to be used based on the name. This looks like you are taking a weighted average of two other commodities not converting from cotton to lint. Consider changing the variable name and definitely expand the comment.